api/oas3.0/v2/device-sdk.yaml

```diff
@@ -0,0 +1,1269 @@
+          additionalProperties:
+            type: array
+            items:
+              type: string
+        service:
+          type: string
+        profile:
+          type: string
+        adminState:
+          type: string
+    ResourceOperation:
+      description: "Defines an operation of which a device is capable."
+      type: object
+      properties:
+        index:
+          type: string
+        operation:
+          type: string
+        deviceResource:
+          type: string
+        parameter:
+          type: string
+        deviceCommand:
+          type: string
+        secondary:
+          type: array
+          items:
+            type: string
+        mappings:
+          type: object
+          additionalProperties:
+            type: string
+    NewSimpleReading:
+      description: "An event reading for a simple data type"
+      allOf:
+        - $ref: '#/components/schemas/NewBaseReading'
+        - type: object
+          properties:
+            floatEncoding:
+              description: "Indicates how a float value is encoded, if the value property contains a float."
+              type: string
+            type:
+              description: "Indicates the datatype of the value property"
+              type: string
+            value:
+              description: "A string representation of the reading's value"
+              type: string
+    SettingRequest:
+      additionalProperties:
+        type: string
+      title: Setting
+      type: object
+      example: {"AHU-TargetTemperature": "28.5", "AHU-TargetBand": "4.0"}
+    UpdateDeviceRequest:
+      allOf:
+        - $ref: '#/components/schemas/BaseRequest'
+      description: "Notification that an existing device associated with this device service has been updated."
+      type: object
+      properties:
+        device:
+          $ref: '#/components/schemas/Device'
+    UpdateProfileRequest:
+      allOf:
+        - $ref: '#/components/schemas/BaseRequest'
+      description: "Notification that an existing device profile has been updated."
+      type: object
+      properties:
+        device:
+          $ref: '#/components/schemas/DeviceProfile'
+    UpdateProvisionWatcherRequest:
+      allOf:
+        - $ref: '#/components/schemas/BaseRequest'
+      description: "Notification that an existing provision watcher associated with this device service has been updated."
+      type: object
+      properties:
+        device:
+          $ref: '#/components/schemas/ProvisionWatcher'
+    VersionRequest:
+      description: "A request associated with the /version endpoint."
+      allOf:
+        - $ref: '#/components/schemas/BaseRequest'
+      type: object
+    VersionResponse:
+      description: "A response returned from the /version endpoint whose purpose is to report out the latest version supported by the service."
+      allOf:
+      - $ref: '#/components/schemas/BaseResponse'
+      type: object
+      properties:
+        version:
```

Comment:
For V1 the SDKs  are returning SDK version and Service version. Need to do the same for V2?

Reply:
Yes, amended